schema/annotation-source.yaml

```diff
@@ -73,12 +73,12 @@ $defs:
         const: Contribution
         default: Contribution
         description: MUST be "Contribution".
-      agent:
+      contributor:
         $ref: "#/$defs/Agent"
       date:
         type: string
         format: date
-      role:
+      contributor_role:
```

Comment:
Changed to more descriptive and precise name used in VA model (makes clear that this field holds the role played by the contributor in the Contribution activity - as opposed to the role played by some other entity).

Reply:
If the contribution is the activity then we should have a 'subtype' or something to specify the precise event that the Contribution represents.